### PR TITLE
fix(serverless): clean up ID token example

### PR DESCRIPTION
**Pull request by ace-n**
~**DO NOT MERGE** until tests pass.~

The system tests are failing on `main` on my local machine, so we should confirm they're working before merging this.

### CODE REVIEW

File: functions/security/idtoken.go [hunk 1]
```diff
@@ -27,14 +26,21 @@ import (
 	"google.golang.org/api/idtoken"
 )
 
-// makeGetRequest makes a request to the provided targetURL with an authenticated client.
-func makeGetRequest(w io.Writer, targetURL string) error {
-	// functionURL := "https://TARGET_URL"
+// `makeGetRequest` makes a request to the provided `targetURL`
```
Comment (codyoss) [inline functions/security/idtoken.go:29]: The func docs should be the thing directly on-top-of the function. Region tags should either be before the docs or after the method.

It looks like this method does not follow our conventions for Go samples. Examples should be in the function body like: https://github.com/GoogleCloudPlatform/golang-samples/blob/main/CONTRIBUTING.md#function-arguments-for-snippets

**Review by codyoss — CHANGES_REQUESTED** [review]
Just a couple of small things

File: functions/security/idtoken.go [hunk 2]
```diff
@@ -27,14 +26,21 @@ import (
...
+// with an authenticated client using audience `audience`.
+//
+// [END functions_bearer_token]
+// Example `audience` value (Cloud Run): https://my-cloud-run-service.run.app/
+// [END cloudrun_service_to_service_auth]
+// [START functions_bearer_token]
+// Example `audience` value (Cloud Functions): https://project-region-projectid.cloudfunctions.net/myFunction
+// [START cloudrun_service_to_service_auth]
+func makeGetRequest(w io.Writer, targetURL string, audience string) error {
```
Comment (muncus) [inline functions/security/idtoken.go:38]: This context may be in the accompanying docs, but in all uses I see here, targetURL and audience are identical. I'd suggest including an explanation of when these args might differ, for other readers without context.

File: functions/security/idtoken_cloudfunction.go [hunk 3]
```diff
@@ -23,13 +23,14 @@ import (
 
 // MakeGetRequestCloudFunction wraps makeGetRequest in an HTTP Cloud Function for system test deployment.
 func MakeGetRequestCloudFunction(w http.ResponseWriter, r *http.Request) {
-	if os.Getenv("TARGET_URL") == "" {
+	url := os.Getenv("TARGET_URL")
```
[the remark this answers is not in the document]
Comment (ace-n) [reply functions/security/idtoken_cloudfunction.go:27]: Fixed.

File: functions/security/idtoken.go (hunk 2 above)
Comment (ace-n) [reply functions/security/idtoken.go:31]: Fixed.

File: functions/security/idtoken.go (hunk 1 above)
Comment (ace-n) [reply functions/security/idtoken.go:29]: Fixed.

**Review by muncus — APPROVED** [review]
I discussed the region_tag shenanigans with Ace offline, and while it is unusual, we do not have a better story for re-use of sample code between samples.

**Review by codyoss — APPROVED** [review]

**Pull request merged**